kafka-connect-transforms/README.md

```diff
@@ -47,3 +47,69 @@ It will promote the `before` or `after` element fields to top level and add the
 | Property            | Description                                                                       |
 |---------------------|-----------------------------------------------------------------------------------|
 | cdc.target.pattern  | Pattern to use for setting the CDC target field value, default is `{db}.{table}`  |
+
+# JsonToMapTransform
```

Comment:
add mention that you need to be using `StringConverter` for the `valueConverter` property since we are parsing json.